### ISSUE TITLE
Story Duration Default - No notification to user who tries to set it higher than 20 seconds

### ISSUE THREAD

**Issue by bmattb** [swissspidy/web-stories-wp#12599]
## Bug Description

Originally reported by @barklund 
 If you set the default interval to a number higher than 20, it’s capped at 20, but with no indication that such a cap exists. Same thing happens inside the editor. We should have some sort of notification & description

## Expected Behaviour

The user should be notified that there is a max value, and that going over that value isn't possible

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by swissspidy** [comment]
Can you please file a separate enhancement request for this? Thanks!

**Comment by merapi** [comment]
> Regarding duration values, it is also possible to set up to 15 decimal digits which is probably not necessary/elegant ![image](https://user-images.githubusercontent.com/3719412/199308232-3d74ab44-1d67-406b-8c02-b0e58796158c.png)
> 
> Cc: @swissspidy

I know it's possible, but do we want decimals at all?
https://amp.dev/documentation/components/amp-story-page/#auto-advance-after-[optional]

**Comment by swissspidy** [comment]
It feels more like an oversight that we allowed it. We can certainly remove that to keep it simple
